src/libs/OptionsListUtils.js

```diff
@@ -277,6 +277,36 @@ function isSearchStringMatch(searchValue, searchText, participantNames = new Set
     });
 }
 
+/**
+ * Returns the given userDetails is currentUser or not.
+ * @param {Object} userDetails
+ * @returns {Bool}
+ */
+
+function isCurrentUser(userDetails) {
+    if (!userDetails) {
+        // If userDetails is null or undefined
+        return false;
+    }
+
+    // If user login is mobile number, append sms domain if not appended already just a fail safe.
+    const userDetailsLogin = addSMSDomainIfPhoneNumber(userDetails.login);
+
+    // Initial check with currentUserLogin
+    let result = currentUserLogin.toLowerCase() === userDetailsLogin.toLowerCase();
+    const loginList = _.isEmpty(currentUser) ? [] : currentUser.loginList;
```

Comment:
I think this can still cause an error:

```
{
  otherParam: true
}
``` 

We check `currentUser.loginList` if `currentUser` is not empty. But it still might not contain `loginList` and would get set to null. This would crash at line 301.

Reply:
Good point. We can check for both like this
 ```js
_.isEmpty(currentUser) ||  _.isEmpty(currentUser.loginList) ? [] : currentUser.loginList ;
```

Let me know what you think.

Reply:
Yep, that seems fine

Reply:
Will update in a minute.